_posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown

```diff
@@ -0,0 +1,114 @@
+---
+title: "<span class='post-title-extra'>Big Discounts</span> When You Shop OpenBazaar This Week"
+layout: post
+date: '2018-11-20 06:30:00 -0300'
+social_title: 'Big Discounts When You Shop OpenBazaar This Week'
+social_description: 'To celebrate the launch of OpenBazaar 2 in November last year we are sharing goodies and offers all month long! Clustered loosely around Black Friday this week are a bunch of exciting offers some vendors have set up just for you.'
+social_image: Pascal_Boyart_on_OpenBazaar.png
+---
+
+To celebrate [the launch of OpenBazaar 2](https://openbazaar.org/blog/openbazaar-20-is-live/) in November of last year we are sharing goodies and offers all month long with our users! Clustered loosely around Black Friday this week are a bunch of exciting offers some OpenBazaar vendors have set up just for you. 
+
+>Unless otherwise specified, use coupon code **OB2Turns1** to claim the discounts on the stores or items described!
+
+#### Here are some great stores and listings for you to shop this week:
+<br>
+#### DrApis
+_75% off_
+
+![DrApis on OpenBazaar](DrApis_on_OpenBazaar.png "DrApis on OpenBazaar")[DrApis on OpenBazaar](https://openbazaar.com/store/QmYTXDyMNjdUSvqNc88T2VeVF3KdG7PMefnGQKrp9NZ5Tp)
+<br>  
+<br>
+DrApis produces and sells premium raw apiculture products from Portugal. Their artisanal raw honey is produced and extracted using exclusively traditional methods, preserving all the natural vitamins, enzymes, phytonutrients and other nutritional elements produced by bees. Now you can enjoy their high-quality products at their lowest prices _ever._ Don't miss this!
+<br>  
+<br>
+#### Matthew Zipkin
+_75% off_
+
+![Matthew Zipkin Bitcoin Clocks on OpenBazaar](Matthew_Zipkin_on_OpenBazaar.png "Matthew Zipkin Bitcoin Clocks on OpenBazaar")[Matthew Zipkin on OpenBazaar](https://openbazaar.com/store/QmeSyTRaNZMD8ajcfbhC8eYibWgnSZtSGUp3Vn59bCnPWC)
+<br>  
+<br>  
+Matthew Zipkin is offering this steep discount on his incredibly neat Bitcoin clocks! has some neat, practical art pieces available in his store. 
+
+Another fun fact, Matthew is also an [OB1 Verified Moderator](https://ob1.io/verified-moderators.html) who is available for you to add to your transactions and provide assistance in case of a dispute. Send him a message and say hello!
+<br>  
+<br>  
+#### CryptoCollectibles
+_75% off_
+
+![CryptoCollectibles on OpenBazaar](CryptoCollectibles_on_OpenBazaar.png "CryptoCollectibles on OpenBazaar")[CryptoCollectibles on OpenBazaar](https://openbazaar.com/store/QmakxNv9Y5YFuAuvjvx4mEda3TmFBaHDtacd1KtEL56mP4/)
+<br>  
+<br> 
+CryptoCollectibles has curated several items to make available at this crazy discount. In honor of his recent passing they are all vintage Stan Lee comics. Some have art by Jack "The King" Kirby, Gene Colan, Gil Kane, and other forefathers of the comic book game. Check out the full list:
```

Comment:
> to make available

That's future tense, presumably they are available now.

I suggest "has curated several items that are available at this"